apps/judicial-system/web/src/routes/Court/RestrictionCase/Confirmation/Confirmation.tsx

```diff
@@ -32,56 +32,91 @@ import {
 } from '@island.is/judicial-system-web/src/graphql/schema'
 import { useCase } from '@island.is/judicial-system-web/src/utils/hooks'
 
+import {
+  JudgeRequestRulingSignatureModal,
+  RegistrarRequestRulingSignatureModal,
+  RulingModifiedModal,
+} from '../../components'
 import { confirmation as strings } from './Confirmation.strings'
 
-type VisibleModal = 'none' | 'rulingModifiedModal' | 'signingModal'
+type VisibleModal =
```

Comment:
How would you feel about defining this type globally instead of in individual components that use the modal? 

Pros: Single declaration of this type
Cons: You could set a modal visible that is not implemented on a particular screen

Just a thought 💭

Reply:
I am not sure. Let's think about it.